applications/StructuralMechanicsApplication/custom_processes/postprocess_eigenvalues_process.cpp

```diff
@@ -0,0 +1,177 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "custom_processes/postprocess_eigenvalues_process.h"
+#include "structural_mechanics_application_variables.h"
+
+
+namespace Kratos
+{
+
+    PostprocessEigenvaluesProcess::PostprocessEigenvaluesProcess(ModelPart &rModelPart,
+                                                                 Parameters OutputParameters) 
```

Comment:
Can this be a reference?

Reply:
Hi Mike, because of how it is done internally, Parameters should be always passed as a copy and not as a reference

Reply:
@RiccardoRossi do you mean in general or only for this class? I thought most classes pass parameters by reference.